courses/fundamentals_of_ada/160_genericity.rst

```diff
@@ -304,7 +307,7 @@ Generic Subprogram Parameters Defaults
       procedure P;
       procedure Callback_1;
```

Comment:
Should probably be renamed to Callback1, unless underscores are ignored for generic `<>` defaults ?

Reply:
Right - I updated the comment, but forgot to update the code